admin/institutions/views.py

```diff
@@ -255,8 +255,9 @@ def form_valid(self, form):
         if not osf_user:
             raise Http404(f'OSF user with id "{user_id}" not found. Please double check.')
 
-        group = Group.objects.filter(name__startswith=f'institution_{target_institution._id}').first()
+        osf_user.add_or_update_affiliated_institution(target_institution)
```

Comment:
Per my discussion with @Johnetordoff , we should not add affiliation to user outside of SSO.